docs/triggerbindings.md

```diff
@@ -10,11 +11,37 @@ kind: TriggerBinding
 metadata:
   name: pipeline-binding
 spec:
-  params:
+  inputParams:
+    - name: message
+      description: The message to print
+      default: This is the default message
+  outputParams:
     - name: gitrevision
       value: $(event.head_commit.id)
     - name: gitrepositoryurl
       value: $(event.repository.url)
+    - name: message
+      value: $(inputParams.message)
 ```
 
 `TriggerBinding`s are connected to `TriggerTemplate`s within an [`EventListener`](eventlisteners.md), which is where the pod is actually instantiated that "listens" for the respective events.
+
+## Input Parameters
+`TriggerBinding`s can declare input parameters that must be supplied to the
```

Comment:
This "must be supplied" portion is sort of my philosophy behind my above comment.

Reply:
I wasn't sure how to word this, because the param is not necessary if there's a `default` value.

Should I rephrase it to:
`TriggerBinding`s can declare input parameters that must be supplied to the
`TriggerBinding` from an `EventListener` if no `default` value is specified.